packages/edit-site/src/components/secondary-sidebar/list-view-sidebar.js

```diff
@@ -71,13 +71,15 @@ export default function ListViewSidebar() {
 				className="edit-site-editor__list-view-panel-content"
 				ref={ useMergeRefs( [ focusReturnRef, focusOnMountRef ] ) }
 			>
-				<BlockNavigationTree
-					blocks={ rootBlocks }
-					selectBlock={ selectEditorBlock }
-					selectedBlockClientId={ selectedBlockClientId }
-					showNestedBlocks
-					__experimentalPersistentListViewFeatures
-				/>
+				<AsyncModeProvider value="true">
```

Comment:
As written, the selectors called in `ListViewSidebar` are synchronous which may be fine but we really want the optimal optimization, we could move the async mode provider up in the tree (around the rendering of `ListViewSidebar` element)